src/common/legacy_config_opts.h

```diff
@@ -961,6 +961,10 @@ OPTION(bluestore_bluefs_balance_interval, OPT_FLOAT) // how often (sec) to balan
 // how often (sec) to dump allocator on allocation failure
 OPTION(bluestore_bluefs_alloc_failure_dump_interval, OPT_FLOAT)
 
+// Enforces db sync with legacy bluefs extents information on close.
+// Enables downgrades to pre-nautilus releases
+OPTION(bluestore_bluefs_db_compatibility, OPT_BOOL)
```

Comment:
I'm not sure if we want this or not.  There are other nautilus changes (the per-pool stats) that won't be backward compatible...

Reply:
Certainly not having this makes things simpler.